### ISSUE TITLE
New Project dialog box has changed

### ISSUE THREAD

**Issue by dmonroym** [dotnet/docs#12983]
Step 3 is no longer true - You can now simply search for the template in VS2019. It would be good to specify if they're targeting .NET Core as well.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 22951aaa-bfb8-f6e5-5d08-d9fd758bc689
* Version Independent ID: 214492ea-f78e-e1b2-5767-ec3cf32cc54a
* Content: [Hello World -- Your first program - C# Programming Guide](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/hello-world-your-first-program)
* Content Source: [docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Good catch. We’ll make sure to get this updated in the .NET Core 3.0 timeframe.
